### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by amynicol1985**

### CODE REVIEW

**Review by jasonblais — CHANGES_REQUESTED** [review]
@amynicol1985 
1. There is an "MLT Planning" section that is not in your Google Doc. Would you like to keep or remove it?
2. For board meetings, there is "30 minute debrief with MLT scheduled same day after board meeting." as a debrief item, which is different in this file

**Comment by amynicol1985** [comment]
MLT Planning is staying as is (for the moment)

Board meeting debrief needs to stay

 

Amy Nicol
Senior Executive Assistant to Ian Tien, CEO of Mattermost, Inc. 
amy.nicol@mattermost.com <mailto:amy.nicol@mattermost.com>  | (650) 667-8512 | mattermost.com <http://mattermost.com/>  

Timezone : Toronto (EST)

File: operations/operations/mlt-cadence/README.md [hunk 1]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
-**8:42am SF time to 8:55am Palo Alto time**
-
-* 8:42am to 8:55am offers 14 minutes to talk through cross-team communication and administrative items outside of meeting
-* Use Zoom in DM channel or phone
-
-## MLT Weekly Tactical
-
-**Tuesdays 10:00am to 11:30am Palo Alto time with MLT**
-The purpose of MLT Weekly Tactical is to keep MLT team on track to quarterly and annual targets. Discussions held in [MLT Weekly Tactical Channel](https://community.mattermost.com/private-core/channels/ijt-weekly-tactical).
+# MLT Cadance
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
# MLT Cadence
```

File: operations/operations/mlt-cadence/README.md [hunk 2]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
+## MLT Weekly Business Review (WBR)
+
+**Tuesdays 10:00am to 11:00am Palo Alto time with MLT**
+
+**Required Attendees - MLT + Extended team (TBD)**
+
+The purpose of MLT WBR is to review the health of the business by reviewing our goals dashboard, identifying key risks/challenges/issues, and identifying owners for actions. Discussions held in [MLT WBR Channel](https://community.mattermost.com/private-core/channels/MLT-WBR).
+
+#### Pre-work
+Dashboard to be updated by all owners by Tuesday 9am Palo Alto Time.
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
* Dashboard to be updated by all owners by Tuesday 9am Palo Alto Time.
```

File: operations/operations/mlt-cadence/README.md [hunk 3]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
+2x2 prepared by each functional owner (TBD if reviewed separately or aggregated into one 2x2) (e.g., next product releases, marketing launches)
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
* 2x2 prepared by each functional owner (TBD if reviewed separately or aggregated into one 2x2) (e.g., next product releases, marketing launches).
```

File: operations/operations/mlt-cadence/README.md [hunk 4]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
-* \(0:45\) **Tactical Agenda -** Discuss queued items, 5m max per topic. Choose:
-  * Discuss now
-  * Discuss later \(another Monthly Strategic or at another Weekly Tactical\)
-  * Discuss 1-1/outside of MLT
-* \(1:10\) **Decisions/Actions** - Summarize decisions/actions in writing
-* \(1:20\) **Cascading Communications** - Decide what to communicate broadly
-  * This includes Customer Obsession Meeting announcements with thank-yous
-  * If meeting ends without agreement, Vice-Chair notes this in [MLT Weekly Tactical channel](https://community.mattermost.com/private-core/channels/ijt-weekly-tactical) with an `@all` mention
+* **\(0:10\) Review WBR Agenda Doc** - Time for all to read data
+* **\(0:15\) MLT Good News -** Always start with anyone wanting to share the good news, e.g., customer wins, new release delivered, new web page launched, etc.
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
* **\(0:15\) MLT Good News:**  Always start with anyone wanting to share the good news, e.g., customer wins, new release delivered, new web page launched, etc.
```

File: operations/operations/mlt-cadence/README.md [hunk 5]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
-  * Check that all objectives have measures and due dates
-* \(0:45\) **Tactical Agenda -** Discuss queued items, 5m max per topic. Choose:
-  * Discuss now
-  * Discuss later \(another Monthly Strategic or at another Weekly Tactical\)
-  * Discuss 1-1/outside of MLT
-* \(1:10\) **Decisions/Actions** - Summarize decisions/actions in writing
-* \(1:20\) **Cascading Communications** - Decide what to communicate broadly
-  * This includes Customer Obsession Meeting announcements with thank-yous
-  * If meeting ends without agreement, Vice-Chair notes this in [MLT Weekly Tactical channel](https://community.mattermost.com/private-core/channels/ijt-weekly-tactical) with an `@all` mention
+* **\(0:10\) Review WBR Agenda Doc** - Time for all to read data
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
* **\(0:10\) Review WBR Agenda Doc:** Time for all to read data
```

File: operations/operations/mlt-cadence/README.md [hunk 6]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
+* **\(0:15\) MLT Good News -** Always start with anyone wanting to share the good news, e.g., customer wins, new release delivered, new web page launched, etc.
+* **\(0:20\) V2MOM Metrics Dashboard Review -** Review Dashboard
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
* **\(0:20\) V2MOM Metrics Dashboard Review:** Review Dashboard
```

File: operations/operations/mlt-cadence/README.md [hunk 7]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
+  * Discuss reds and new goals to create
+  * Discuss yellows and paths to green
+  * Review key updates to greens
+  * Create new action items and assign to owners in Asana 
+* **\(0:25\) 2X2 Business Review -** Review WBR 2x2 document summarizing:
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
* **\(0:25\) 2X2 Business Review:** Review WBR 2x2 document summarizing:  
```

File: operations/operations/mlt-cadence/README.md [hunk 8]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
+    * Big updates from Sales, R&D, Marketing, Finance, etc.
+  * Issues/Challenges/Risks
+    * Key customer issues (at-risk customers, deals, etc.)
+    * Key R&D risks/challenges/issues
+    * Key marketing risks/challenges
+  * Goals
+  * Upcoming Milestones
+  * Upcoming R&D milestones and dates
+  * Upcoming marketing milestones and dates
+* **\(0:55\) Decisions/Actions -** Summarize decisions/actions in writing
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
* **\(0:55\) Decisions/Actions:** Summarize decisions/actions in writing 
```

File: operations/operations/mlt-cadence/README.md [hunk 9]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
+* **\(0:55\) Decisions/Actions -** Summarize decisions/actions in writing 
+
+## MLT Weekly Executive Meeting 
+
+**Tuesdays 11:00am to 12:00pm Palo Alto time with MLT**
+
+**Required Attendees - MLT only + HR Representative**
 
-## MLT Monthly Strategic
+The purpose of MLT Staff is to discuss MLT topics like CEO updates, board-level topics, hiring, sensitive HR-related announcements, company-wide programs like annual reviews & compensation, staff top of mind, etc.
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
The purpose of the MLT Weekly Executive Meeting is to discuss MLT topics like CEO updates, board-level topics, hiring, sensitive HR-related announcements, company-wide programs like annual reviews and compensation, staff top of mind, etc.
```

File: operations/operations/mlt-cadence/README.md [hunk 10]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
 
-**Pre-work**
+## MLT Weekly Deep Dive Meeting
 
-* \(Vice-Chair\) Agenda drafted based on queued topics in [MLT Monthly Strategic channel](https://community.mattermost.com/private-core/channels/monthly-strategic-mtg)
-* \(Vice-Chair\) Agenda posted in MLT Monthly Strategic [Scratch document](https://docs.google.com/document/d/1O6ok1Uvb8uuCCFaxYjkkPZCPfDFK8nlb7OGrmyfwtCc/edit)
-* \(MLT\) People who queue topics post relevant materials in Agenda 24 hours before meeting
+**Wednesdays 9:00am to 10:30am Palo Alto time with MLT**
+
+**Required Attendees - MLT only + extended team (as appropriate by topic)  **
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md:77]:
```suggestion
**Required Attendees - MLT only + extended team (as appropriate by topic) **
```

File: operations/operations/mlt-cadence/README.md [hunk 11]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
+
+The purpose of this new MLT weekly deep dive series is to achieve a deeper understanding and clarity on functional areas and/or strategic topics through reviews, discussions, and decisions.
+
+Topics are queued in a [central spreadsheet](https://docs.google.com/spreadsheets/d/1d_kBiMSsrjYVWX6JUjzNLruq8yQciIdIz-Vtmy69ygo/edit#gid=0) and prioritized by the Vice-Chair. Examples of topics are Cloud MVP, DevOps command center launch, customer journey & packaging proposals, marketing OKRs, etc.
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
Topics are queued in a [central spreadsheet](https://docs.google.com/spreadsheets/d/1d_kBiMSsrjYVWX6JUjzNLruq8yQciIdIz-Vtmy69ygo/edit#gid=0) and prioritized by the Vice-Chair. Examples of topics are Cloud MVP, DevOps Command Center launch, customer journey and packaging proposals, marketing OKRs, etc.
```

File: operations/operations/mlt-cadence/README.md [hunk 12]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
+
+#### Pre-work
+
+* (Vice-Chair) Agenda drafted based on queued topics in a central google docs sheet
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
* (Vice-Chair) Agenda drafted based on queued topics in a central Google Doc
```

File: operations/operations/mlt-cadence/README.md [hunk 13]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
+* (Owner) Prepare narrative document for the meeting
+
+#### Process
+
+* 2 slots of 45 minutes each; Typically 2 topics for 2 slots but bigger topics can occupy 2 slots.
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
* Two slots of 45 minutes each. Typically two topics - one for each slot. However, bigger topics can occupy two slots
```

File: operations/operations/mlt-cadence/README.md [hunk 14]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
+* MLT reviews document for the first 15 minutes; discusses for the next 25 minutes to reach any decisions for clarity and closure.
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
* MLT reviews document for the first 15 minutes; discusses for the next 25 minutes to reach any decisions for clarity and closure
```

File: operations/operations/mlt-cadence/README.md [hunk 15]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
+* (5m before end of slot) Decisions/actions are summarized
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
* (Five minutes before end of slot) Decisions/actions are summarized
```

File: operations/operations/mlt-cadence/README.md [hunk 16]
```diff
@@ -239,7 +121,7 @@ We’re currently focused on quarterly plans and will move to halves.
 
 The following should happen within a 2 week period, with 1-2 iterations in each step:
 
-1. CEO discusses company and department VPMOMs 1-1 with department heads to ensure alignment
+1. CEO discusses company and department V2MOMs 1-1 with department heads to ensure alignment
    * If applicable during the period, sales, CEO and finance set or adjust the revenue targets
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
   * If applicable during the period, Sales, CEO, and Finance set or adjust the revenue targets
```

File: operations/operations/mlt-cadence/README.md [hunk 17]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
 **T-Minus 2 Weeks to announcement.**
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
**T-Minus 2 Weeks to announcement**
```

File: operations/operations/mlt-cadence/README.md [hunk 18]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
...
   * Draft agenda
     * Review of MLT Plan Process
-    * Company VPMOM & Q2 plan
-    * Sales VPMOM, Q2 plan and proposed org
-    * R&D VPMOM, Q2 plan and proposed org
-    * Marketing VPMOM, Q2 plan and proposed org
-    * Customer Success VPMOM, Q2 plan and proposed org
-    * Finance VPMOM, Q2 plan and proposed org
-    * People VPMOM, Q2 plan and proposed org
+    * Company V2MOM & Q2 plan
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
    * Company V2MOM and Q2 plan
```

File: operations/operations/mlt-cadence/README.md [hunk 19]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
...
+    * Sales V2MOM, Q2 plan and proposed org
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
    * Sales V2MOM, Q2 plan, and proposed org
```

File: operations/operations/mlt-cadence/README.md [hunk 20]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
...
+    * R&D V2MOM, Q2 plan and proposed org
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
    * R&D V2MOM, Q2 plan, and proposed org
```

File: operations/operations/mlt-cadence/README.md [hunk 21]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
...
+    * Marketing V2MOM, Q2 plan and proposed org
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
    * Marketing V2MOM, Q2 plan, and proposed org
```

File: operations/operations/mlt-cadence/README.md [hunk 22]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
...
+    * Customer Success V2MOM, Q2 plan and proposed org
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
    * Customer Success V2MOM, Q2 plan, and proposed org
```

File: operations/operations/mlt-cadence/README.md [hunk 23]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
...
+    * Finance V2MOM, Q2 plan and proposed org
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
    * Finance V2MOM, Q2 plan, and proposed org
```

File: operations/operations/mlt-cadence/README.md [hunk 24]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
...
+    * People V2MOM, Q2 plan and proposed org
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
    * People V2MOM, Q2 plan, and proposed org
```

File: operations/operations/mlt-cadence/README.md [hunk 25]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
...
 * MLT socializes updates to high-level plan with their departments
 
 **Finalize Plan**
 
 **T-Minus 1 Week to announcement**
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
**T-Minus 1 week to announcement**
```

File: operations/operations/mlt-cadence/README.md [hunk 26]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
...
+    * Company V2MOM & Q2 plan
+    * Sales V2MOM, Q2 plan and proposed org
+    * R&D V2MOM, Q2 plan and proposed org
+    * Marketing V2MOM, Q2 plan and proposed org
+    * Customer Success V2MOM, Q2 plan and proposed org
+    * Finance V2MOM, Q2 plan and proposed org
+    * People V2MOM, Q2 plan and proposed org
 * MLT socializes updates to high-level plan with their departments
 
 **Finalize Plan**
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
**Finalize plan**
```

File: operations/operations/mlt-cadence/README.md [hunk 27]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
...
 
 #### Quarterly Business Review
 
-QBR is an exercise to increase output through concise, efficient review of quarterly goals for company and department in the context of VPMOMs.
+QBR is an exercise to increase output through concise, efficient review of quarterly goals for company and department in the context of V2MOMs.
 
 **Time and People**
 
-* Scheduled monthly for 3.5 hours with Mattermost Leadership Team.
+* Scheduled 1 week after quarter end and can be broken out into
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
* Scheduled 1 week after quarter end and can be broken out into:
```

File: operations/operations/mlt-cadence/README.md [hunk 28]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
...
-
-TBD
+Each quarter we review process towards our V2MOMs and discuss achievements and opportunities.
 
 #### Quarterly Business Review
 
-QBR is an exercise to increase output through concise, efficient review of quarterly goals for company and department in the context of VPMOMs.
+QBR is an exercise to increase output through concise, efficient review of quarterly goals for company and department in the context of V2MOMs.
 
 **Time and People**
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
**Time and people**
```

File: operations/operations/mlt-cadence/README.md [hunk 29]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
...
 
-* Scheduled monthly for 3.5 hours with Mattermost Leadership Team.
+* Scheduled 1 week after quarter end and can be broken out into
+    * 90 mins for Sales QBR (30 mins for each region)   
+    * 45 mins CS QBR
+    * 2 hours for MLT QBR. (30 mins each for Product, Engineering, Q&A & Marketing)
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
    * 2 hours for MLT QBR (30 mins each for Product, Engineering, G&A, and Marketing) 
```

File: operations/operations/mlt-cadence/README.md [hunk 30]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
...
 **Time and People**
 
-* Scheduled monthly for 3.5 hours with Mattermost Leadership Team.
+* Scheduled 1 week after quarter end and can be broken out into
+    * 90 mins for Sales QBR (30 mins for each region)   
+    * 45 mins CS QBR
+    * 2 hours for MLT QBR. (30 mins each for Product, Engineering, Q&A & Marketing) 
+
 
 **Prior to Meeting**
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
**Prior to meeting**
```

File: operations/operations/mlt-cadence/README.md [hunk 31]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
...
 
 * Materials for QBR agenda are shared by Friday prior to the QBR meeting.
 
 **Agenda**
 
-* \(CEO\) Reviews [MLT Cadence VPMOM](http://handbook.mattermost.com/leadership/leadership-meetings.html#MLT-Cadence-VPMOM) \(10 minutes\)
-* \(Each department head\) Reviews quarterly goals in the context of VPMOMs, including the following \(25 minutes per person, including 5 minutes for review, 5 minutes for MLT to share written feedback, and 15 minutes for discussion\):
+* \(CEO\) Reviews [MLT Cadence V2MOM](http://handbook.mattermost.com/leadership/leadership-meetings.html#MLT-Cadence-VPMOM) \(10 minutes\)
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
* \(CEO\) Reviews [MLT Cadence V2MOM](https://handbook.mattermost.com/leadership/leadership-meetings.html#MLT-Cadence-VPMOM) \(10 minutes\)
```

File: operations/operations/mlt-cadence/README.md [hunk 32]
```diff
@@ -366,33 +236,33 @@ Typically there is not pressure for budget or headcount discussions in Q3, focus
 **Q3 Punchlist for Next Fiscal Year Planning**
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md:229]:
```suggestion
**Q3 Punchlist for next fiscal year planning**
```

File: operations/operations/mlt-cadence/README.md [hunk 33]
```diff
@@ -366,33 +236,33 @@ Typically there is not pressure for budget or headcount discussions in Q3, focus
...
 
 1. MLT and middle managers work through key Obstacles for next fiscal year
 2. Finance works with MLT to draft financials plan for next fiscal year
-3. CEO works with MLT to develop MLT VPMOM from 50% to 99%
-4. Department heads work with their teams, peer stakeholders and CEO to develop Department VPMOM from 50% to 99%
-5. 99% MLT VPMOM is reviewed by MLT and agreed
-6. 99% Department VPMOMs are reviewed by MLT peers and CEO and agreed
-7. Plan for next fiscal year is locked based on VPMOMs
+3. CEO works with MLT to develop MLT V2MOM from 50% to 99%
+4. Department heads work with their teams, peer stakeholders and CEO to develop Department V2MOM from 50% to 99%
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
4. Department heads work with their teams, peer stakeholders, and CEO to develop Department V2MOM from 50% to 99%
```

File: operations/operations/mlt-cadence/README.md [hunk 34]
```diff
@@ -366,33 +236,33 @@ Typically there is not pressure for budget or headcount discussions in Q3, focus
...
+5. 99% MLT V2MOM is reviewed by MLT and agreed
+6. 99% Department V2MOMs are reviewed by MLT peers and CEO and agreed
+7. Plan for next fiscal year is locked based on V2MOMs
 
-#### MLT VPMOM Review Process
+#### MLT V2MOM Review Process
 
-VPMOMs, quarterly plans and proposed orgs are reviewed by MLT. Use the following order to review VPMOMs:
+V2MOMs, quarterly plans and proposed orgs are reviewed by MLT. Use the following order to review V2MOMs:
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
V2MOMs, quarterly plans, and proposed orgs are reviewed by MLT. Use the following order to review V2MOMs:
```

File: operations/operations/mlt-cadence/README.md [hunk 35]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
+  * Upcoming Milestones
+  * Upcoming R&D milestones and dates
+  * Upcoming marketing milestones and dates
+* **\(0:55\) Decisions/Actions -** Summarize decisions/actions in writing 
+
+## MLT Weekly Executive Meeting 
+
+**Tuesdays 11:00am to 12:00pm Palo Alto time with MLT**
+
+**Required Attendees - MLT only + HR Representative**
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
**Required attendees: MLT only + HR representative**
```

File: operations/operations/mlt-cadence/README.md [hunk 36]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
-
-**Tuesdays 10:00am to 11:30am Palo Alto time with MLT**
-The purpose of MLT Weekly Tactical is to keep MLT team on track to quarterly and annual targets. Discussions held in [MLT Weekly Tactical Channel](https://community.mattermost.com/private-core/channels/ijt-weekly-tactical).
+# MLT Cadance
+
+## MLT Weekly Business Review (WBR)
+
+**Tuesdays 10:00am to 11:00am Palo Alto time with MLT**
+
+**Required Attendees - MLT + Extended team (TBD)**
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
**Required attendees: MLT + extended team (TBD)**
```

File: operations/operations/mlt-cadence/README.md (hunk 10 above)
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
**Required attendees: MLT only + extended team (as appropriate by topic)**
```

File: operations/operations/mlt-cadence/README.md [hunk 37]
```diff
@@ -22,194 +22,76 @@ This section outlines:
...
+## MLT Weekly Deep Dive Meeting
 
-* \(Vice-Chair\) Agenda drafted based on queued topics in [MLT Monthly Strategic channel](https://community.mattermost.com/private-core/channels/monthly-strategic-mtg)
-* \(Vice-Chair\) Agenda posted in MLT Monthly Strategic [Scratch document](https://docs.google.com/document/d/1O6ok1Uvb8uuCCFaxYjkkPZCPfDFK8nlb7OGrmyfwtCc/edit)
-* \(MLT\) People who queue topics post relevant materials in Agenda 24 hours before meeting
+**Wednesdays 9:00am to 10:30am Palo Alto time with MLT**
+
+**Required Attendees - MLT only + extended team (as appropriate by topic)  **
+
+The purpose of this new MLT weekly deep dive series is to achieve a deeper understanding and clarity on functional areas and/or strategic topics through reviews, discussions, and decisions.
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]:
```suggestion
The purpose of this new MLT Weekly Deep Dive series is to achieve a deeper understanding and clarity on functional areas and/or strategic topics through reviews, discussions, and decisions.
```

File: operations/operations/mlt-cadence/README.md [hunk 38]
```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
...
 ## Quarterly Business Review \(QBR\)
 
-Each quarter we review process towards our VPMOMs and discuss achievements and opportunities.
-
-#### QBR VPMOM
-
-TBD
+Each quarter we review process towards our V2MOMs and discuss achievements and opportunities.
 
 #### Quarterly Business Review
```
Comment (justinegeffen) [inline operations/operations/mlt-cadence/README.md]: I think this may be extraneous as there is already a QBR heading (H2) above it.

**Comment by justinegeffen** [comment]
@amynicol1985, I've made one or two comments that may need your feedback. If you have the bandwidth to review all my suggestions to ensure nothing has been incorrectly changed that'd be awesome otherwise I can commit them and do a second pass.

File: operations/operations/mlt-cadence/README.md (hunk 38 above)
Comment (amynicol1985) [reply operations/operations/mlt-cadence/README.md]: I agree. We can remove the duplication.

**Comment by amynicol1985** [comment]
@justinegeffen - can you please merge your changes? For some reason, the full list is partially hidden for me. 
image

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thanks, @amynicol1985! I've committed all the changes and removed the extraneous header.

**Review by jasonblais — APPROVED** [review]
I didn't review line-by-line, but content looks good. Approving. Thanks Amy!

**Pull request merged**